src/http/status.cr

```diff
@@ -74,10 +74,10 @@ enum HTTP::Status
   # ```
   # require "http/status"
   #
-  # HTTP::Status.new(100)  # => CONTINUE
-  # HTTP::Status.new(202)  # => ACCEPTED
-  # HTTP::Status.new(123)  # => 123
-  # HTTP::Status.new(1000) # raises ArgumentError
+  # HTTP::Status.new(100).to_s # => "CONTINUE"
```

Comment:
This change seems similarly unnecessary when using the inspect output (https://github.com/crystal-lang/crystal/pull/9818#discussion_r515400631)

Reply:
ditto